cypress/support/index.js

```diff
@@ -19,12 +19,6 @@ require('./commands/storefront-api-commands');
 // Import fixture commands.js
 require('./commands/system-commands');
 
-// Import themes:
-if (Cypress.config('useDarkTheme')) {
```

Comment:
cypress-dark was removed

Reply:
Thanks ✔️  It doesn't affect our custom Shopware theme, tight? Iirc it is an own implementation.

Reply:
Unfortunately this change isn't enough to remove `cypress-dark`. In the command `activateShopwareTheme` it still relies on finding the stylesheet for it and inject the shopware theme. Therefore the file `cypress/support/system-commands.js` has to be adjusted like this:

```js
Cypress.Commands.add('activateShopwareTheme', () => {
    const $head = Cypress.$(parent.window.document.head); // eslint-disable-line no-restricted-globals

    $head.append(
        `<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/brands.css" integrity="sha384-i2PyM6FMpVnxjRPi0KW/xIS7hkeSznkllv+Hx/MtYDaHA5VcF0yL3KVlvzp8bWjQ" crossorigin="anonymous">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/fontawesome.css" integrity="sha384-sri+NftO+0hcisDKgr287Y/1LVnInHJ1l+XC7+FOabmTTIK0HnE2ID+xxvJ21c5J" crossorigin="anonymous">
<link rel="stylesheet" href="https://gitcdn.xyz/repo/shopware/e2e-testsuite/master/theme/shopware.css" crossorigin="anonymous">`
    );
});
```

Reply:
Please be careful to not kill our own Shopware Theme here. It's oriented on the `cypress-dark`theme, but an own implementation I'd like to keep.

Reply:
I'm well aware of this. The Shopware theme isn't injected properly when the cypres-dark stylesheet wasn't found. This actually ensures our theme is still present and working :+1:

Reply:
The theme does not seem to work even after changing this.

Reply:
Ok, that's strange. I don't know if I'll get the time to inspect that though 🙈  if we cannot keep our own theme due to `m cypress-dark`, so be it. @klarstil as you did this in branch 3.x, do you have an idea?

Reply:
nvm, It was an issue on my side